lineman/app/components/group_page/group_page.haml

```diff
@@ -34,3 +34,4 @@
         %membership_requests_card{group: 'groupPage.group'}
         %members_card{group: 'groupPage.group'}
         %subgroups_card{group: 'groupPage.group'}
+        %group_previous_proposals_card{group: 'groupPage.group'}
```

Comment:
Ah, so I think the way to do this cleanly is to have 

- a method on the group_model, which gives the closed proposals (you have this already I think)
- a `previous_proposal_collection` directive, which accepts a series of proposals
- a `previous_proposal` collection, which accepts a proposal

Similar to the `thread_preview_collection` and `thread_preview` directives

```
%proposals_collection{proposals: 'group.closedProposals()'}
```
```
%proposals_collection{proposals: 'discussion.nonActiveProposals()'}
```

Then, the proposal_collection.haml looks like
```
.thread-proposal-collection{ng-repeat: 'proposal in proposals'}
```